### PR TITLE
[release/7.0-rc1] Register authz middleware for Blazor Server IndAuth

### PR DESCRIPTION
**Pull request by captainsafia**
Addresses https://github.com/aspnet/AspNetCore-ManualTests/issues/1542

Follow-up to https://github.com/dotnet/aspnetcore/pull/43167

### CODE REVIEW

**Review by HaoK — APPROVED** [review]

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.Main.cs [hunk 1]
```diff
@@ -153,6 +153,11 @@ public static void Main(string[] args)
 
         app.UseRouting();
 
+        #if (IndividualAuth)
+        app.UseAuthorization();
+
+        #endif
+
```
Comment (adityamandaleeka) [inline src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.Main.cs:160]: Will this add an extra newline?

Comment (captainsafia) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.Main.cs:160]: Yeah, I believe you're right. It didn't previously have it.

https://github.com/dotnet/aspnetcore/blob/9ac46165aa66079292ef32b2fbfebfb8717fd6e1/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.Main.cs#L156-L161

```suggestion
```

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.cs [hunk 2]
```diff
@@ -147,6 +147,11 @@
 
 app.UseRouting();
 
+#if (IndividualAuth)
+app.UseAuthorization();
+
+#endif
+
```
Comment (captainsafia) [inline src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Program.cs:154]:
```suggestion
```

**Review by adityamandaleeka — APPROVED** [review]

**Comment by captainsafia** [comment]
@dotnet/aspnet-admins Can I get help merging?

**Comment by dougbu** [comment]
Sure, I can help. But has this change been approved for RC1❔ It doesn't look like an obvious `tell-mode` change.

**Comment by adityamandaleeka** [comment]
@dougbu I'm acting approver while Kevin's out. This is good to go for RC1.

**Comment by dougbu** [comment]
Got it. Thanks @adityamandaleeka for the info.

**Pull request merged**